src/service/resources-impl.js

```diff
@@ -1659,11 +1659,15 @@ export class ResourcesImpl {
   /**
    * @param {!./task-queue.TaskDef} task
    * @param {boolean} success
-   * @param {*=} opt_reason
+   * @param {!AbortSignal|*} data
    * @return {!Promise|undefined}
    * @private
    */
-  taskComplete_(task, success, opt_reason) {
+  taskComplete_(task, success, data) {
```

Comment:
Ditto: I'd expect still a rejection reason here. Except that it should be a `AbortError`.